replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorHistoryManagerImpl.java

```diff
@@ -102,34 +103,38 @@ public void init() {
   }
 
   @Override
-  public List<ReplicationStatus> getReplicationEvents() {
+  public Stream<ReplicationStatus> objects() {
     return helper.getTypeForFilter(
         filterBuilder
             .attribute(Core.METACARD_TAGS)
             .is()
             .equalTo()
             .text(ReplicationHistory.METACARD_TAG),
-        this::getStatusFromMetacard);
+        this::getStatusFromMetacard).stream();
   }
 
   @Override
-  public List<ReplicationStatus> getReplicationEvents(String replicatorid) {
+  public Stream<ReplicationStatus> getByReplicatorId(String replicatorid) {
```

Comment:
✏️ methods that returned `Stream<>` should not be prefixed with `get` by Java convention